rclpy/test/test_node.py

```diff
@@ -550,12 +508,66 @@ def test_node_has_parameter_services(self):
         )
 
 
+class TestExecutor(unittest.TestCase):
```

Comment:
this could move to `test_executor.py`? but it looks like `test_executor.py` tests executor implementation, but here does interaction from node instance. so probably we can ignore this comment.

Reply:
Yeah, that one is a bit subjective.  I could see it going into either file.  For the sake of not making this PR any bigger, I'm going to leave this where it is; we can always move it later.

Thanks for the review!